<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_UI</class>
 <widget class="QMainWindow" name="MainWindow_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>743</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>en_message</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gui_resource.qrc">
    <normaloff>:/gui/resources/image/icon.svg</normaloff>:/gui/resources/image/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_Key_Info">
      <property name="title">
       <string>Key info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QLineEdit" name="LineEdit_Key_Text">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>key</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="comboBox_Key_Mode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentText">
          <string notr="true">Text</string>
         </property>
         <item>
          <property name="text">
           <string>Text</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="Button_Load_Key">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="Label_Encrypt_Mode">
         <property name="text">
          <string>Encrypt algorithm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="Button_Save_Key">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="comboBox_Encrypt_Algorithm">
         <item>
          <property name="text">
           <string>AES-256-CFB</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget_Crypto_Data">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Crypto_Text_Tab">
       <attribute name="title">
        <string>Text</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QPushButton" name="Button_Cipher_Text_Copy">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_Text_Decrypt_Window" stretch="9,1">
          <item>
           <widget class="QTextEdit" name="TextEdit_Cipher_Text">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Button_Text_Up">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Up</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="Button_Raw_Text_Clear">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="Button_Raw_Text_Copy">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="Button_Raw_Text_Paste">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Paste</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_Raw_Text_Window" stretch="9,1">
          <item>
           <widget class="QTextEdit" name="TextEdit_Raw_Text"/>
          </item>
          <item>
           <widget class="QPushButton" name="Button_Text_Down">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Down</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout_Encrypt_Button">
          <item>
           <widget class="QPushButton" name="Button_Encrypt_Text">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Encrypt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Button_Decrypt_Text">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Decrypt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="Button_Cipher_Text_Paste">
          <property name="text">
           <string>Paste</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="Button_Cipher_Text_Clear">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Crypto_File_Tab">
       <attribute name="title">
        <string>File</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_File_Path">
          <item row="0" column="0">
           <widget class="QLabel" name="Label_Source_File_Path">
            <property name="text">
             <string>Source Path</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="LineEdit_Source_File_Path"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="Button_Source_File_Browse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="Label_Output_File_Path">
            <property name="text">
             <string>Output Path</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="LineEdit_Output_File_Path"/>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="Button_Output_File_Browse">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_File_Tab_Encrypt_Button">
          <item>
           <widget class="QPushButton" name="Button_Encrypt_File">
            <property name="text">
             <string>Encrypt</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Button_Decrypt_File">
            <property name="text">
             <string>Decrypt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Key">
    <property name="title">
     <string>KEY</string>
    </property>
    <addaction name="actionExchange_Key"/>
    <addaction name="actionRandom_Text_Key"/>
   </widget>
   <addaction name="menu_Key"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExchange_Key">
   <property name="text">
    <string>Exchange Key</string>
   </property>
  </action>
  <action name="actionRandom_Text_Key">
   <property name="text">
    <string>Random Text Key</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>Button_Decrypt_Text</tabstop>
 </tabstops>
 <resources>
  <include location="gui_resource.qrc"/>
 </resources>
 <connections/>
</ui>
